proxy/envoy/config.go

```diff
@@ -92,6 +89,13 @@ func buildConfig(listeners Listeners, clusters Clusters, lds bool, mesh *proxyco
 		StatsdUDPIPAddress: mesh.StatsdUdpAddress,
 	}
 
+	if mesh.ProxyAdminPort > 0 {
+		out.Admin = &Admin{
+			AccessLogPath: DefaultAccessLog,
+			Address:       fmt.Sprintf("tcp://%s:%d", WildcardAddress, mesh.ProxyAdminPort),
```

Comment:
If you want to mess with the admin port config - maybe make the entire address an option ( so we can specify 
tcp://127.0.0.1:.. to avoid wide open admin interface).

Same for proxy port, now that I think about it we must bind it to 127.0.0.1 by default.